ansible/openrailwaymap/openrailwaymap.yml

```diff
@@ -0,0 +1,98 @@
+# SPDX-License-Identifier: MIT
+- hosts: letsencrypt
+  become: yes
+  tags: letsencrypt
+  tasks:
+    - import_role:
+        name: roles/letsencrypt
```

Comment:
Given that the CI complains, have you missed a ```git add```.

Reply:
The files were added to Git but the GitHub action called Ansible-Lint the wrong way (now fixed).